exclip/exclip.c

```diff
@@ -21,17 +21,27 @@
  *    min if val is lower than min, max if val is greater than max, 
  *        and otherwise val
  */
-int clip(int val, int min, int max)
+signed int clip(signed int val,signed int min,signed int max)
 {
-    /* You should modify clip to meet the specifications */
-    return 0;
+if(max>min)
```

Comment:
Observe proper code whitespace and indentations.

`  if (max > min) {`

Reply:
i am newbie.